requirements.txt

```diff
@@ -1 +1,3 @@
-ipython >= 7.5.0
\ No newline at end of file
+pytest
```

Comment:
How about listing requirements to .github/workflows/tests.yml?
requirements.txt connotes a python package, which this isn't
I'd prefer to hide this detail from the user.

Reply:
moved to .requirements.txt instead.